app/controllers/solidus_paypal_commerce_platform/orders_controller.rb

```diff
@@ -10,7 +10,7 @@ def create
       authorize! :create, ::Spree::Order
 
       @order = ::Spree::Order.create!(
-        user: try_spree_current_user,
+        user: ::Spree.solidus_gem_version > Gem::Version.new('3.2') ? spree_current_user : try_spree_current_user,
```

Comment:
nit: I think this is supposed to be `>=` although it will work in practice because the latest has `.2` as the patch version and is technically above `3.2`

Reply:
good catch, My bad for over looking